src/Analyzers/CSharp/Tests/AddParameter/AddParameterTests.cs

```diff
@@ -3030,13 +3032,15 @@ record struct Test(string V);
         [Fact]
         public async Task TestNamingConventions_PrimaryConstructor_Struct()
         {
-            await TestInRegularAndScript1Async(@"[|new Test(""repro"")|];
+            await TestInRegularAndScript1Async("""
+                [|new Test("repro")|];
 
-struct Test();
-", @"new Test(""repro"");
+                struct Test();
+                """, """
+                new Test("repro");
 
-struct Test(string v);
-");
+                struct Test(string v);
+                """);
```

Comment:
This pull request removes the first/final newline from inputs, which causes most tests to no longer represent the majority case that occurs in real code. It also makes it impossible to review this to try and identify cases that do/do not matter for the change in input semantics.

Reply:
Honestly, it is just not that relevant.

Reply:
Talked with Cyrus offline and he's verified that the input containing a newline at beginning/end isn't relevant to any of these tests. Other than that, this looks good to me and I'll go ahead and sign off.